Dockerfile

```diff
@@ -21,5 +21,5 @@ ENV NODE_ENV=production
 ENV NODE_CONFIG_DIR=/data/config
 
 CMD ["bot"]
-ENTRYPOINT ["./mjolnir-entrypoint.sh"]
+ENTRYPOINT ["./draupnir-entrypoint.sh"]
```

Comment:
I'd prefer to be conservative about changing this, just in case someone else has built their own image based on our own.

Reply:
You do make a fair point about this. But i also do think that we should break this at some point as we should remove all references to mjolnir from the codebase that are not historical references.

As in if we refer to mjolnir it should be like for example refering to a Mjolnir issue or that this file integrates work from mjolnir.

Reply:
> But i also do think that we should break this at some point as we should remove all references to mjolnir from the codebase that are not historical references.

Why?

Reply:
it gives a certain vibe to some people that we refer to our selfs as mjolnir all over the place. This PR should hopefully reduce the chances of end users seeing us call our selfs mjolnir in logs or via command errors.